extensions/agroal/deployment/src/test/java/io/quarkus/agroal/test/AgroalVaultITCase.java

```diff
@@ -0,0 +1,54 @@
+package io.quarkus.agroal.test;
+
+import static io.quarkus.vault.VaultTestExtension.testDataSource;
+
+import java.sql.SQLException;
+
+import javax.inject.Inject;
+
+import org.jboss.logging.Logger;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.agroal.api.AgroalDataSource;
+import io.quarkus.agroal.DataSource;
+import io.quarkus.test.QuarkusUnitTest;
+import io.quarkus.vault.VaultTestExtension;
+
+@DisabledOnOs(OS.WINDOWS)
```

Comment:
I'm not very comfortable having a feature that is not tested at all on Windows. What's the issue you have?

Reply:
me neither ;)

apparently docker is not available on the windows machine where the ci is being done.
see https://github.com/quarkusio/quarkus/pull/2897#issuecomment-505883787
I also created https://github.com/quarkusio/quarkus/issues/3796